src/models/pie.js

```diff
@@ -338,6 +338,38 @@ nv.models.pie = function() {
                         }
                         return label;
                     })
+                    .each(function (d, i) {
+                        if (!this.getBBox) return;
+                        var bb = this.getBBox(),
```

Comment:
iirc, getBBox throws errors if the chart is hidden or detached from the dom or something... I remember we had to replace it with getBoundingClientRect in another location a while back.